app/Labels/LabelServiceProvider.php

```diff
@@ -0,0 +1,360 @@
+            CreateLabelRequestHandler::class,
+            PatchLabelRequestHandler::class,
+            GetLabelRequestHandler::class,
+            SearchLabelsRequestHandler::class,
+            self::JSON_READ_REPOSITORY,
+            self::JSON_WRITE_REPOSITORY,
+            self::RELATIONS_WRITE_REPOSITORY,
+            self::RELATIONS_READ_REPOSITORY,
+            self::LABEL_ROLES_WRITE_REPOSITORY,
+            'labels.constraint_aware_service',
+            CdbXMLToJsonLDLabelImporter::class,
+            self::UNIQUE_EVENT_STORE,
+            self::REPOSITORY,
+            self::COMMAND_HANDLER,
+            self::JSON_PROJECTOR,
+            self::RELATIONS_PROJECTOR,
+            self::LABEL_ROLES_PROJECTOR,
+            LabelVisibilityOnRelatedDocumentsProjector::class,
+            self::QUERY_FACTORY,
+            self::LOGGER,
+        ];
+    }
+
+    public function register(): void
+    {
+        $container = $this->getContainer();
+
+        $this->setUpLogger($container);
+
+        $this->setUpReadModels($container);
+
+        $this->setUpServices($container);
+
+        $this->setUpEventStore($container);
+
+        $this->setUpCommandHandler($container);
+
+        $this->setUpProjectors($container);
+
+        $this->setUpQueryFactory($container);
+
+        $container->addShared(
+            CreateLabelRequestHandler::class,
+            function () use ($container): CreateLabelRequestHandler {
+                return new CreateLabelRequestHandler(
+                    $container->get('event_command_bus'),
+                    new Version4Generator()
+                );
+            }
+        );
+
+        $container->addShared(
+            PatchLabelRequestHandler::class,
+            function () use ($container): PatchLabelRequestHandler {
+                return new PatchLabelRequestHandler($container->get('event_command_bus'));
+            }
+        );
+
+        $container->addShared(
+            GetLabelRequestHandler::class,
+            function () use ($container): GetLabelRequestHandler {
+                return new GetLabelRequestHandler($container->get(LabelServiceProvider::JSON_READ_REPOSITORY));
+            }
+        );
+
+        $container->addShared(
+            SearchLabelsRequestHandler::class,
+            function () use ($container): SearchLabelsRequestHandler {
+                return new SearchLabelsRequestHandler(
+                    $container->get(LabelServiceProvider::JSON_READ_REPOSITORY),
+                    $container->get(LabelServiceProvider::QUERY_FACTORY)
+                );
+            }
+        );
+    }
+
+    private function setUpReadModels(Container $container): void
+    {
+        $container->addShared(
+            self::JSON_READ_REPOSITORY,
+            function () use ($container): ReadRepositoryInterface {
+                $labels = file_exists(__DIR__ . '/../../../config.excluded_labels.php') ? require __DIR__ . '/../../../config.excluded_labels.php' : [];
```

Comment:
```suggestion
                $labels = file_exists(__DIR__ . '/../../config.excluded_labels.php') ? require __DIR__ . '/../../config.excluded_labels.php' : [];
```
File location has changed so this path needs to change as well